recipe/meta.yaml

```diff
@@ -1,37 +1,99 @@
-{% set unix_version = "5.26.2" %}
+{% set unix_version = "5.32.1" %}
 {% set win_version = unix_version + ".1" %}
+{% set major_minor_version = unix_version.split(".")[:2] | join(".") %}
 
 package:
   name: perl
   version: {{ unix_version }}  # [not win]
   version: {{ win_version }}   # [win]
 
 source:
-  url: http://www.cpan.org/src/5.0/perl-{{ unix_version }}.tar.gz                                                 # [unix]
-  url: http://strawberryperl.com/download/{{ win_version }}/strawberry-perl-{{ win_version }}-64bit-portable.zip  # [win64]
-  url: http://strawberryperl.com/download/{{ win_version }}/strawberry-perl-{{ win_version }}-32bit-portable.zip  # [win32]
-  sha256: 572f9cea625d6062f8a63b5cee9d3ee840800a001d2bb201a41b9a177ab7f70d                                        # [unix]
-  # sha1 hashes published at: http://strawberryperl.com/releases.html
-  sha1: 7996609176451c54864f700ffa299ee81712d8bd                                                                  # [win64]
-  sha1: 73564a7cef397c4e0da93612c00fa5f482e652b3                                                                  # [win32]
+  - url: https://www.cpan.org/src/5.0/perl-{{ unix_version }}.tar.xz                                                 # [unix]
+    sha256: 57cc47c735c8300a8ce2fa0643507b44c4ae59012bfdad0121313db639e02309                                        # [unix]
+    patches:                                                                                                        # [unix]
+      - 0001-Use-userelocatableinc-together-with-useshrplib.patch                                                   # [unix]
+  - url: https://strawberryperl.com/download/{{ win_version }}/strawberry-perl-{{ win_version }}-64bit-portable.zip  # [win64]
+    url: https://strawberryperl.com/download/{{ win_version }}/strawberry-perl-{{ win_version }}-32bit-portable.zip  # [win32]
```

Comment:
Do we need to keep around the win32 hashes anymore?

Reply:
We shouldn't. I went ahead and removed those. Thanks!